src/main/java/org/folio/search/cql/builders/ExactTermQueryBuilder.java

```diff
@@ -25,7 +26,10 @@ public class ExactTermQueryBuilder extends FulltextQueryBuilder {
 
   @Override
   public QueryBuilder getQuery(Object term, String resource, String... fields) {
-    return multiMatchQuery(term, fields).type(PHRASE);
+    List<String> updatedFields = Arrays.stream(fields)
+      .map(field -> updatePathForTermQueries(resource, field))
```

Comment:
This adjustment have to be done only when we have `/string` modifier. `==` operator must work as before. Changes should be only with `==\string` operator with modifier. 
The same like we have there:
![image](https://github.com/folio-org/mod-search/assets/5904625/126b6ea5-f9f9-4f9e-bf59-d2605249b164)